config/backend-config/backend-config.go

```diff
@@ -208,18 +208,30 @@ func newForDeployment(deploymentType deployment.Type, configEnvHandler types.Con
 				eb:               pubsub.New(),
 			},
 		}
+	// DEPRECATED: MultiTenantType should be used instead
```

Comment:
Let's get rid of this code. WDYT?

Reply:
I will merge this and open a new PR to remove hosted type code.